README.md

```diff
@@ -1,20 +1,21 @@
 # react-native-snackbar
-Implementation of the material design snacker component. See [Google Design](https://www.google.com/design/spec/components/snackbars-toasts.html#snackbars-toasts-usage)
 
+Implementation of the material design Snackbar component for both Android and iOS. See [Google Design](https://material.google.com/components/snackbars-toasts.html) for more info.
 
 ## Installation
+
 To use this extension, you have to do the following steps:
 
-1. In your react-native project, run 
+1. In your react-native project, run
 	```
 	npm install react-native-snackbar
```

Comment:
Both way works, and PR was submit 2 month ago. But RN community is really hooked to `yarn`.